src/ebonite/ext/pandas/dataset_source.py

```diff
@@ -191,16 +191,17 @@ class PandasReader(DatasetReader):
     """DatasetReader for pandas dataframes
 
     :param format: PandasFormat instance to use
-    :param data_type: DataFrameType to use for aliging read data
+    :param dataset_type: DataFrameType to use for aliging read data
     """
 
-    def __init__(self, format: PandasFormat, data_type: DataFrameType):
-        self.data_type = data_type
+    def __init__(self, format: PandasFormat, dataset_type: DataFrameType, path: str = None):
+        super(PandasReader, self).__init__(dataset_type)
+        self.path = path or PANDAS_DATA_FILE
```

Comment:
What if dataset is in remote location, or in any sort of container? How that will work?